packages/v4/patternfly-docs/content/design-guidelines/usage-and-behavior/bulk-selection/bulk-selection.md

```diff
@@ -4,7 +4,7 @@ section: patterns
 ---
 
 ## Bulk selection
-Use a bulk selection pattern when you want to select or deselect multiple items in a content view (list, table, or card grid). The bulk selector uses a [Split button](/components/dropdown/react#split-button) component to control selection from the [Toolbar](/components/toolbar). Besides controlling selection, the bulk selector reflects the selection status of the related component (partially selected, all items selected, or no items selected).
+Use a bulk selection pattern when you want to select or deselect multiple items in a content view (list, table, or card grid). The bulk selector uses a [Split button](/components/menus/dropdown/html) component to control selection from the [Toolbar](/components/toolbar). Besides controlling selection, the bulk selector reflects the selection status of the related component (partially selected, all items selected, or no items selected).
```

Comment:
In this case, we'd advise them to build a split button using the MenuToggle. And the url for that is `/components/menus/menu-toggle#split-button-toggle-with-checkbox`